### DISCUSSION TITLE
Clarification on Django App Example

### DISCUSSION THREAD

**Discussion by nikoder** [geopython/pygeoapi#1265] [Q&A]
Dear PyGeoAPI Team and community,

I hope you're doing well. I have a question regarding the Django example in the `0.15.0` tag and would appreciate your guidance.

In the [README.md](https://github.com/geopython/pygeoapi/tree/0.15.0/examples/django/sample_project), specifically in the `urls.py` imports, it seems that the example expects a `pygeoapi.django_pygeoapi` Python package. However, when I installed the `0.15.0` tag using pip, I noticed that it only includes a module named `pygeoapi.django_`. Consequently, I'm encountering difficulties in getting the demo to work as expected, particularly under Django 4.

Could you clarify if my assumption that [`pygeoapi.django_`](https://github.com/geopython/pygeoapi/tree/0.15.0/pygeoapi/django_) is the package referred to in the example is correct? Alternatively, is it possible that the example is outdated or incomplete?

I greatly appreciate your assistance. Thank you for your time and for your valuable contributions to the PyGeoAPI project - I'm really glad that you're opening up the OGC API world to the Python ecosystem!

**Comment by francbartoli** [comment]
Hi @nikoder, can you open an issue with a basic example on how to reproduce it? Thanks in advance for reporting
